SteamBot/Logging/Log.cs

```diff
@@ -95,10 +99,22 @@ public void RemoveLoggingObject(LoggerBase logger)
 
         public void Dispose()
         {
-            Disposed = true;
-            foreach (IDisposable logger in LoggerObjects.OfType<IDisposable>())
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        public void Dispose(bool disposing)
+        {
+            if (!Disposed)
             {
-                logger.Dispose();
+                if (disposing)
+                {
+                    foreach (IDisposable disposableLogger in LoggerObjects.OfType<IDisposable>())
+                        disposableLogger.Dispose();
```

Comment:
@BlueRaja Is it alright for me to assume Logging instances are a Managed Resource?